Makefile

```diff
@@ -1,7 +1,8 @@
+.PHONY: all install
 all: install
 	grunt build
 install:
-	git submodule update --init --recursive
+	git submodule update --init --recursive --remote
```

Comment:
Please revert this change, because it's not always a good idea to use the latest submodule state.